app/assets/javascripts/modules/FilterTable.js

```diff
@@ -65,6 +65,18 @@ define(['jquery', 'DoughBaseComponent', 'List', 'ListFuzzySearch'],
     return $.unique(filterClasses);
   };
 
+  /**
+   * enableFuzzySearch
+   *
+   * Find input field with class filterFieldClass then add fuzzy-search class to it.
+   *
+   */
+  FilterTableProto.enableFuzzySearch = function () {
+    this.$el.find('.' + this.config.filterFieldClass).each(function(_, cell) {
```

Comment:
This could be done as a one liner, as in this should do the trick:

```
this.$el.find('.' + this.config.filterFieldClass).addClass('fuzzy-search');
```